css/src/atomics/colors.scss

```diff
@@ -167,6 +171,17 @@
 	background-color: $body-background-accent !important;
 }
 
+.background-color-body-accent-onhover {
```

Comment:
Should this class specify an `outline-color`?

Reply:
good catch, added!